WS2012R2/lisa/remote-scripts/ica/perf_ntttcp_client.sh

```diff
@@ -510,28 +510,135 @@ if [ -d  $HOME/$log_folder ]; then
     rm -rf $HOME/$log_folder
 fi
 
-mkdir $HOME/$log_folder
-# Determine test type lagscope/ntttcp
-if [[ "${TEST_TYPE:-UNDEFINED}" = "UNDEFINED" ]] || [[ "${TEST_TYPE}" = "ntttcp" ]]; then
+mkdir  "$HOME/$log_folder"
+# Determine test type lagscope/ntttcp/ntttcp-udp
+if [[ "${UDP_MODE}" = "yes" ]]; then
     eth_log="$HOME/$log_folder/eth_report.log"
-    echo "#test_connections    throughput_gbps    average_packet_size" > $eth_log
+    echo "#test_connections    throughput_gbps    average_packet_size" > "$eth_log"
```

Comment:
never use "$eth_log", always use "${eth_log}". this applies for all strings with vars